ui/page/main_page.go

```diff
@@ -407,7 +407,13 @@ func (mp *MainPage) HandleUserInteractions() {
 				continue
 			}
 
-			mp.ChangeFragment(pg)
+			if !mp.WL.MultiWallet.IsSynced() {
+				mp.ChangeFragment(pg)
+			} else if mp.WL.MultiWallet.IsSyncing() {
+				mp.Toast.Notify("Wallet is syncing, please wait")
+			} else {
+				mp.Toast.Notify("Wallet not synced")
```

Comment:
```suggestion
				mp.Toast.Notify("Not connected to the Decred network")
```
<img src="https://user-images.githubusercontent.com/25265396/165132808-d775418c-18d3-4815-a8b2-8596f159584c.jpg" width="300"/>

Reply:
My concern with this is what happens after you are connected, and you try to access the page?

Reply:
`if !mp.WL.MultiWallet.IsSynced() {
				mp.ChangeFragment(pg)`

It goes to the page like on the code, my suggestion was just to change the text, from "wallet not synced"

Reply:
alright.. lets get this implemented and we move from there.